components/kyma-environment-broker/internal/hyperscaler/azure/config.go

```diff
@@ -43,3 +43,75 @@ func (c *Config) Environment() (*azure.Environment, error) {
 
 	return c.environment, nil
 }
+
+func azureRegions() map[string]bool {
+	return map[string]bool{
+		"australiacentral":   true,
+		"australiacentral2":  true,
+		"australiaeast":      true,
+		"australiasoutheast": true,
+		"brazilsouth":        true,
+		"canadacentral":      true,
+		"canadaeast":         true,
+		"centralindia":       true,
+		"centralus":          true,
+		"eastasia":           true,
+		"eastus":             true,
+		"eastus2":            true,
+		"francecentral":      true,
+		"francesouth":        true,
+		"germanynorth":       true,
+		"germanywestcentral": true,
+		"japaneast":          true,
+		"japanwest":          true,
+		"koreacentral":       true,
+		"koreasouth":         true,
+		"northcentralus":     true,
+		"northeurope":        true,
+		"norwayeast":         true,
+		"norwaywest":         true,
+		"southafricanorth":   true,
+		"southafricawest":    true,
+		"southcentralus":     true,
+		"southeastasia":      true,
+		"southindia":         true,
+		"switzerlandnorth":   true,
+		"switzerlandwest":    true,
+		"uaecentral":         true,
+		"uaenorth":           true,
+		"uksouth":            true,
+		"ukwest":             true,
+		"westcentralus":      true,
+		"westeurope":         true,
+		"westindia":          true,
+		"westus":             true,
+		"westus2":            true,
+	}
+}
+
+func gcpToAzureRegionMapping() map[string]string {
+	return map[string]string{
```

Comment:
could you also adjust this enum in plan schema: https://github.com/kyma-incubator/compass/blob/master/components/kyma-environment-broker/internal/broker/plans.go#L63

so just put there your region list from this map:
```
               "enum": ["asia-east1", "asia-east2", "asia-northeast1", "asia-northeast2", "asia-northeast3", "asia-south1", "asia-southeast1", "australia-southeast1", "europe-north1", "europe-west1", "europe-west2", "europe-west3", "europe-west4", "europe-west6", "northamerica-northeast1", "southamerica-east1", "us-central1", "us-east1", "us-east4", "us-west1", "us-west2", "us-west3"]
```

as a result it will be unified, and we also have validation for this plan schema so in `mapRegion` you will don't get anything unexpected :)

Reply:
Be aware the region list (in plans.go) must be valid from Gardener point of view - Gardener must accept the region, so we cannot add an existing GCP region which is not supported by Gardener.

Reply:
so it would be safer to use the list from the schema in my code?

Reply:
yes, you can use the list from the schema. please, add there (in the schema) a comment regarding usage of region so in the future if someone adds a new region - the gcpToAzureRegionMapping function could be changed.

Reply:
will do! 👍